bfdd/bfd.c

```diff
@@ -751,6 +775,14 @@ int ptm_bfd_sess_del(struct bfd_peer_cfg *bpc)
 
 	/* This pointer is being referenced, don't let it be deleted. */
 	if (bs->refcount > 0) {
+		if (NULL != ebuf) {
```

Comment:
Do you plan to use this somewhere? You've changed the function signature but you don't use it.